### PR TITLE
Hotfix/344 icons don't appear

### PR DESCRIPTION
**Pull request by weronikaolejniczak**
- I added MaterialCommunityIcons.ttf to Gradle build settings, that way they are in the build and render in the application.
- I changed all icons I could find to MaterialCommunityIcons.
- I corrected the syntax in some places.

### CODE REVIEW

File: src/domains/notes/containers/addNote/AddNote.js [hunk 1]
```diff
@@ -122,16 +122,16 @@ const AddNote = (props) => {
...
+      label: 'Diary',
+      value: 'Diary',
       color: '#FF4500',
       fontWeight: 'bold',
     },
   ];
 
   const placeholder = {
     label: 'Select a category...',
-    value: 'Categoryless',
```
Comment (CalderBriar) [inline src/domains/notes/containers/addNote/AddNote.js:134]: It was a thing? Oh my..

File: src/domains/transport/components/item/Transport.js [hunk 2]
```diff
@@ -62,12 +61,12 @@ const Transport = (props) => {
   const pickPDF = async () => {
     try {
       const res = await DocumentPicker.pick({
-        type: [DocumentPicker.types.pdf],
+        type: [DocumentPicker.types.PDF],
```
Comment (CalderBriar) [inline src/domains/transport/components/item/Transport.js:64]: Please undo this change. Documentation of Document Picker says: 
"DocumentPicker.types.*
DocumentPicker.types.* provides a few common types for use as type values, these types will use the correct format for each platform (MIME types on Android, UTIs on iOS).

DocumentPicker.types.allFiles: All document types, on Android this is */*, on iOS is public.content (note that some binary and archive types do not inherit from public.content)
DocumentPicker.types.images: All image types (image/* or public.image)
DocumentPicker.types.plainText: Plain text files ie: .txt (text/plain or public.plain-text)
DocumentPicker.types.audio: All audio types (audio/* or public.audio)
DocumentPicker.types.pdf: PDF documents (application/pdf or com.adobe.pdf)
DocumentPicker.types.zip: Zip files (application/zip or public.zip-archive)
DocumentPicker.types.csv: Csv files (text/csv or public.comma-separated-values-text)"

**Review by CalderBriar — CHANGES_REQUESTED** [review]
Please undo the change in src/domains/transport/components/item/Transport.js
image
Documentation of Document Picker says:
"DocumentPicker.types.*
DocumentPicker.types.* provides a few common types for use as type values, these types will use the correct format for each platform (MIME types on Android, UTIs on iOS).

DocumentPicker.types.allFiles: All document types, on Android this is /, on iOS is public.content (note that some binary and archive types do not inherit from public.content)
DocumentPicker.types.images: All image types (image/* or public.image)
DocumentPicker.types.plainText: Plain text files ie: .txt (text/plain or public.plain-text)
DocumentPicker.types.audio: All audio types (audio/* or public.audio)
DocumentPicker.types.pdf: PDF documents (application/pdf or com.adobe.pdf)
DocumentPicker.types.zip: Zip files (application/zip or public.zip-archive)
DocumentPicker.types.csv: Csv files (text/csv or public.comma-separated-values-text)"

File: src/domains/notes/containers/addNote/AddNote.js (hunk 1 above)
Comment (weronikaolejniczak) [reply src/domains/notes/containers/addNote/AddNote.js:134]: Yup, that was before. Changed it to "Without category" in one of the commits.

File: src/domains/transport/components/item/Transport.js (hunk 2 above)
Comment (weronikaolejniczak) [reply src/domains/transport/components/item/Transport.js:64]: K, on it.

**Review by mkowalczyk97 — APPROVED** [review]
Approved

**Pull request merged**